auth/services/oauth/oauth.go

```diff
@@ -243,7 +243,17 @@ func (s *service) validateLegalBase(jwtBearerToken *services.NutsJwtBearerToken)
 	return nil
 }
 
-// CreateJwtBearerToken creates a JwtBearerToken from the given CreateJwtGrantRequest
+// GetOAuthEndpointURL returns the oauth2 endpoint URL of the custodian for a service
+func (s *service) GetOAuthEndpointURL(service string, custodian did.DID) (string, error) {
+	_, endpointURL, err := services.ResolveCompoundServiceURL(s.docResolver, custodian, service, services.OAuthEndpointType, nil)
+	if err != nil {
+		return "", err
```

Comment:
might be good to wrap it in something like `unable to resolve OAuth endpoint URL: %w` to give it some context

Reply:
Fixed